app/point.py

```diff
@@ -26,3 +27,12 @@ def x(self) -> float:
     @property
     def y(self) -> float:
         return self._y
+
+
+d = Dictionary()
+d[Point(1, 2)] = 9
+d[Point(5, 2)] = 13
+print(d[Point(5, 2)])
+print(hash(d[Point(1, 2)]))
+print(hash(Point(4, 2)))
+print(len(d))
```

Comment:
This testing should be removed from point.py file, because any time, the point is being imported - this code will be executed. If you want to test something - better to do it in main.py, using something like `if __name__ == "__main__":`